packages/slice-machine/pages/onboarding.tsx

```diff
@@ -61,32 +68,32 @@ const WelcomeSlide = ({ onClick }: { onClick: () => void }) => (
     </Button>
   </>
 );
-const BuildSlicesSlide = () => (
+const BuildSlicesSlide = ({ onEnded }: { onEnded: () => void }) => (
```

Comment:
Maybe you can create a SlicesSlideProps type for { onEnded: () => void } and using it in the 3 slides ?

Reply:
maybe, could call it something like `WithOnEnded`